packages/components/FocusZone/macos/RCTFocusZone.m

```diff
@@ -453,14 +458,18 @@ - (void)keyDown:(NSEvent *)event
 
 	BOOL passthrough = NO;
 	NSView *viewToFocus = nil;
-
+	BOOL noOp = NO;
 	if ([self disabled] || action == FocusZoneActionNone)
```

Comment:
The variable doesnt seem very descriptive

Reply:
Hmm, adding documentation should clarify things up?

Reply:
Let me know if there's any suggestions on renaming this variable

Reply:
Heh... I prefer the opposite so that the default value of a bool is false. I'm willing to go whatever way makes like easier though :D

Reply:
plot twist: no more boolean 🙃